options/posix/include/sys/types.h

```diff
@@ -22,6 +22,7 @@
 
 #include <bits/posix/fsblkcnt_t.h>
 #include <bits/posix/fsfilcnt_t.h>
+#include <sys/select.h>
```

Comment:
This looks strange. Which types do we actually want? Those types should be extracted into a `bits/` header (see the other includes above this one); that header should be included here.